getting-started.adoc

```diff
@@ -81,11 +79,13 @@ image::after-adding-fiat-account.png[Configuring a national currency account]
 
 There's just one more thing to do before we can trade with Bisq: we need to send enough bitcoin to our Bisq wallet to cover a security deposit as well as trading and mining fees.
 
-To prevent fraud, Bisq requires both buyers and sellers to post a security deposit in bitcoin for every trade. On the buyer's side, this amount is usually {btc-deposit} BTC, plus a trading fee and mining fees for the Bitcoin transactions involved in each trade.
+Security deposits help prevent fraud, so Bisq requires both buyers and sellers to post one (in bitcoin) for every trade. On the buyer's side, this is usually 10% of the trade amount.
 
-As of this writing, that means you'll need {btc-deposit-plus-fees} BTC.
+NOTE: Sellers can specify their preferred deposit percentage when creating an offer to sell bitcoin. The default is 10%, but it's possible you'll come across offers that require bigger (or smaller) security deposits.
 
-If you don't already own this much bitcoin, you'll need to obtain it. How? You've got options. Friends and family are often the best option—simply buy a small amount of bitcoin directly from someone you trust. Otherwise, you can try a https://coinatmradar.com/[local bitcoin ATM] (many don't require ID for small transactions), https://www.meetup.com/topics/bitcoin/[bitcoin meetups/events], or vouchers.
+As of this writing, you would need 0.0012 BTC to take an offer to buy 0.01 BTC that requires a 10% security deposit. But please note that mining fees can fluctuate, some sellers may require bigger security deposits, and you'll need more for bigger trades, so it may be better to acquire more than the bare minimum so you have more flexibility.
+
+If you don't already own the bitcoin you need, you'll need to obtain it. How? You've got options. Friends and family are often the best option—simply buy a small amount of bitcoin directly from someone you trust. Otherwise, you can try a https://coinatmradar.com/[local bitcoin ATM] (many don't require ID for small transactions), https://www.meetup.com/topics/bitcoin/[bitcoin meetups/events], or vouchers.
```

Comment:
`Friends and family are often the best option—simply buy a small amount of bitcoin directly from someone you trust.` This sounds a bit unrealistic at the present state of the marker IMO (How many of us have family members who actively buy and sell BTC?). Maybe better to substute this text with something more generic, omitting *Friends and family are often the best option*.

Reply:
Friends/family is merely a suggestion. Appreciate the thought, but will leave it in since it was already there.